esx_service/utils/auth_api.py

```diff
@@ -432,13 +454,24 @@ def _tenant_vm_add(name, vm_list):
         error_info = error_code.generate_error_info(ErrorCode.VM_ALREADY_IN_TENANT, vm_list, name)
         return error_info
 
+    error_info, tenant_list = get_tenant_list_from_db()
```

Comment:
I think we should return an error - we probably need to introduce a new error code here.